docroot/sites/wellbeing.uiowa.edu/modules/wellbeing_core/wellbeing_core.module

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for wellbeing_core module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function wellbeing_core_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id === 'node_page_edit_form') {
```

Comment:
Turns out it was actually because I forgot to enable the wellbeing_core module and add it to the site split. But this change was necessary too, thanks for catching it!